doc/userguide/configuration/exception-policies.rst

```diff
@@ -200,6 +216,48 @@ Notes:
    * Not valid means that Suricata will error out and won't start.
    * ``REJECT`` will make Suricata send a Reset-packet unreach error to the sender of the matching packet.
 
+.. _eps_stats:
+
+Available Stats
+---------------
```

Comment:
this gets rejected in CI here https://github.com/OISF/suricata/actions/runs/16999224830/job/48197275037?pr=13724

Reply:
I think the title level should be `*` instead of `-`